lib/api/db/index.ts

```diff
@@ -51,19 +51,21 @@ export const setUpDB = async () => {
 
 export const connect = async () => {
   if (!db) {
-    if (config.database.type === DATABASES.FIREBASE) {
-       logger('INFO', 'Connecting to firebase database')
-      db = await connectFirebase()
-    } else if (config.database.type === DATABASES.MONGO) {
-       logger('INFO', 'Connecting to Mongo Database')
-      db = await connectMongoDB()
-    } else {
-      logger('INFO', 'Using in memory database')
-      db = connectInMemoryDB()
-    }
+    switch (config.database.type) {
+      case DATABASES.MONGO: {
+        logger('INFO', 'Connecting to Mongo Database')
 
-    await setUpDB()
+        db = await connectMongoDB()
+        await setUpMongoDB()
+
+        break
+      }
+      default: {
+        logger('INFO', 'Using in memory database')
+        db = connectInMemoryDB()
```

Comment:
we need to call setUpDb also in the "in memory" database, to pre-seed with some mock data

Reply:
does "in memory" database has the same api as MongoDB? I mean collections and etc